torchinductor/codegen/wrapper.py

```diff
@@ -70,6 +70,10 @@ def __init__(self):
                 aten = torch.ops.aten
             """
         )
+        if config.triton.use_mm:
+            self.header.writeline(
+                "from torchinductor.triton_ops.matmul import matmul_out as triton_mm_out"
```

Comment:
Not sure if my comments or not. 
But is it possible to use 
"import triton
import triton.ops.matmul as triton_mm_out"
to call the triton matmul directly rather than putting another copy of triton.ops.matmul in torchinductor directory. Or does the import module has to be inside torchinductor?

Reply:
Hi Yunjie, yes directly importing `triton.ops.matmul` would work too. The thing here is that we created a `out` variant of the original triton mm implementation, and we may need to add some additional arguments to do the template approach. Basically we'd like to do some customization of the original implementation.

Reply:
That makes sense. Thanks for the explanation.